### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and Adding Extensions _ScreenReader:Screen Reader is not announcing the complete information about the "Installed" and "Recommended" extensions.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141718]
GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Narrator;#NVDA;  #ScreenReader; #WCAG4.2.1; #AINotInScope; #FTP; #STP;

Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

Repro Steps:
1)Turn on Narrtor/NVDA
2)Open Visual studio code editor.3).Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3)Using Tab key navigate to the "More" option and select different options( "Show installed Extension, Show outdated extension, 
   Show popular Extension,Etc")
4)Select an extension and using "Tab" and "Arrow" keys navigate through the extension details window on the RHS
5)Using tab key navigate to different extension for "Installation/Uninstallation".
6)Verify whether the screen reader is announcing the complete information or not for "Installed" and "Recommended".

Actual Result:
Screen Reader is not announcing the complete information about the "Installed" and "Recommended" extensions.

Expected Result:
Screen Reader should announce the complete information about the "Installed" and "Recommended" extensions.

User Impact:
Screen reader user will face difficulty as the complete information is not being announced.
